🔧 Missing formatEventDate export in calendar utils

## Problem
`EventCalendar` component imports `formatEventDate` from `@/utils/calendar` but function doesn't exist or isn't exported.

## Error
```
src/components/EventCalendar.tsx(18,3): error TS2305: Module '@/utils/calendar' has no exported member 'formatEventDate'.
```

## Impact
- Severity: MEDIUM - P2
- Type-checking fails
- Calendar component broken
- /calendar page non-functional

## Fix Options
1. Export the function if it exists
2. Implement the function
3. Use existing formatDate from utils/format.ts

## Priority
P2 - Calendar feature broken.

--- a/apps/web/src/utils/calendar.ts
+++ b/apps/web/src/utils/calendar.ts
@@ -15,6 +15,23 @@ export function timestampToDate(timestamp: bigint): Date {
   return new Date(Number(timestamp) * 1000);
 }
 
+export function formatEventDate(timestamp: bigint, language: LanguageCode = 'en'): string {
+  const date = timestampToDate(timestamp);
+  try {
+    return new Intl.DateTimeFormat(language, {
+      weekday: 'short',
+      year: 'numeric',
+      month: 'short',
+      day: '2-digit',
+      hour: '2-digit',
+      minute: '2-digit',
+      hour12: false,
+    }).format(date);
+  } catch {
+    return format(date, "EEE, MMM dd, yyyy, HH:mm");
+  }
+}
+
 /**
  * Generate ICS file content for calendar export
  * Compatible with Google Calendar, Apple Calendar, Outlook

